.travis.yml

```diff
@@ -2,6 +2,9 @@ language: java
 
 sudo: false
 
+before_install:
+  - export M2_HOME=/usr/local/maven-3.3.9
+
```

Comment:
Thanks for clarification.
Please add this to the documentation PR that `M2_HOME` must be specified.